nms/app/packages/magmalte/app/views/equipment/GatewayDetailMain.js

```diff
@@ -360,27 +364,47 @@ function GatewayOverview() {
           </Grid>
         </Grid>
         <Grid item xs={12} md={6}>
-          <Grid container spacing={4} direction="column">
-            <Grid item>
-              <CardTitleRow icon={GraphicEqIcon} label="Status" />
-              <GatewayDetailStatus />
-            </Grid>
-            <Grid item>
-              <CardTitleRow
-                icon={SettingsInputAntennaIcon}
-                label="Connected eNodeBs"
-              />
-              <GatewayDetailEnodebs gwInfo={gwInfo} />
-            </Grid>
-            <Grid item>
-              <CardTitleRow icon={PeopleIcon} label="Subscribers" />
-              <GatewayDetailSubscribers gwInfo={gwInfo} />
-            </Grid>
-          </Grid>
+          <GatewayInfo />
         </Grid>
       </Grid>
     </div>
   );
 }
 
+function GatewayInfo() {
+  const {match} = useRouter();
+  const networkId: string = nullthrows(match.params.networkId);
+  const gatewayId: string = nullthrows(match.params.gatewayId);
+  // Auto refresh gateways every 30 seconds
+  const state = useRefreshingContext({
```

Comment:
We seem to be missing the autoRefresh checkbox. Can you also add that in here. 
We also have to add refresh functionality for GatewayDetailEnodebs(for refreshing eNodeB state) and GatewayDetailSubscribers(subscriber state), We want to do both to be default, not autorefreshing. Did you plan to make that as a separate PR ?